relay-server/src/statsd.rs

```diff
@@ -316,6 +316,8 @@ pub enum RelayTimers {
     OutcomeAggregatorFlushTime,
     /// Time in milliseconds spent on parsing, normalizing and scrubbing replay recordings.
     ReplayRecordingProcessing,
+    /// Total time spent to send a request to upstream and handle the response.
```

Comment:
Looking at the code, it's not really true. This only count the send request and receive the response, but not handle it.